bot/__main__.py

```diff
@@ -6,13 +6,13 @@
 from common.utils.setup import load_data
 
 from bot.utils.add_cython_ext import add_cython_ext
-from bot.utils.setup import load_secrets, load_translations
+from bot.utils.setup import load_disabled_translations, load_secrets, load_translations
 from bot.villager_bot import VillagerBotCluster
 
 
 async def main_async() -> None:
     secrets = load_secrets()
-    translations = load_translations()
+    translations = load_translations(load_disabled_translations())
```

Comment:
Instead of making and using `load_disabled_translations`, why not just use `load_secrets()`?